frank-flow/src/frontend/src/app/flow/canvas/canvas.component.ts

```diff
@@ -37,19 +39,22 @@ export class CanvasComponent implements AfterViewInit, OnDestroy {
 
   jsPlumbInstance!: jsPlumbInstance;
   currentFileSubscription!: Subscription;
+  settingsSubscription!: Subscription;
   flowUpdate = false;
   locked!: boolean;
```

Comment:
We should be consistent with making variables private.

Reply:
We should ;)